src/test/java/fi/metatavu/metaform/test/functional/tests/MetaformTestsIT.java

```diff
@@ -29,13 +29,14 @@
   @Test
   public void testCreateMetaform() throws Exception {
```

Comment:
Added tests for succesfull slug update, conflicting slug update and null slug update